src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/FormInspector.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import {computed} from 'mobx';
-import type {IObservableValue} from 'mobx';
+import type {IObservableValue} from 'mobx/lib/mobx';
```

Comment:
flow does not recognize the declaration file (`mobx.js.flow`) wen using `'mobx'` 😕